matryoshka.tmux

```diff
@@ -15,8 +15,26 @@ if [ -z "$up_recursive_keybind" ]; then up_recursive_keybind='F3'; fi
 inactive_status_style="$(tmux show-option -gqv @matryoshka_inactive_status_style)"
 if [ -z "$inactive_status_style" ]; then inactive_status_style='fg=colour245,bg=colour238'; fi
 
+inactive_status_update_mode="$(tmux show-option -gqv @matryoshka_inactive_status_style_update_mode)"
+if [ -z "$inactive_status_update_mode" ]; then inactive_status_update_mode='override'; fi
+
 status_style_option="$(tmux show-option -gqv @matryoshka_status_style_option)"
 if [ -z "$status_style_option" ]; then status_style_option='status-style'; fi
+
+inactive_status_style_replacements="$(tmux show-option -gqv @matryoshka_inactive_status_style_replacements)"
+if [ "$inactive_status_update_mode" = "replace" ]; then
+    original_style="$(tmux show-option -gqv $status_style_option)"
+    inactive_status_style="$original_style"
+
+    IFS=';' read -ra PAIRS <<< "$inactive_status_style_replacements"  # Split by semicolon
+	for pair in "${PAIRS[@]}"; do
+        IFS=',' read -ra REPLACE <<< "$pair"  # Split each pair by comma
+        before="${REPLACE[0]}"
+        after="${REPLACE[1]}"
+        inactive_status_style="${inactive_status_style//${before}/${after}}"  # Replace substrings
+    done
+fi
```

Comment:
This logic implies that a snapshot of the status style is captured when loading the plugin to compute the alternative inactive version, right? This could lead to discrepancies if another plugin, or even the user, later changes the active status style before deactivating the instance.

For example, if the status style is originally `<blue>time-and-date</blue>` and we source matryoshka with replacements `blue,gray`, we will get the inactive status style `<gray>time-and-date</gray>`. Now, if another plugin gets sourced changing the status style to `<blue>my-status-style</blue>`, the inactive status style will still be `<gray>time-and-date</gray>`, which is suboptimal.

Have the tmux shortcut compute the inactive status style on the fly would address this, and I think it's worth implementing. What do you think?